tests/core/test_db.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+import pytest_asyncio
+
+from lnbits.db import QueryValues
+
+
+@pytest_asyncio.fixture
+async def json_test_table(db):
+    try:
+        await db.execute("CREATE TABLE test (data JSON)")
+        yield db
+    finally:
+        await db.execute("DROP TABLE test")
+
+
+@pytest.mark.asyncio
+async def test_sql_json(db, json_test_table):
+    obj = {"a": 3, "b": "bar", "c": {"nested": "d"}}
+    await db.execute("INSERT INTO test VALUES(?)", (obj,))
+
+    values = QueryValues()
+    row = await db.fetchone(
+        f"""
+        SELECT * FROM test WHERE {values.json_path('data', 'a', type_=int)} >= {values(obj["a"])}
+        """,
+        values,
+    )
+    assert row.data == obj
+
+    values = QueryValues()
+    row = await db.fetchone(
+        f"SELECT * FROM test WHERE {values.json_path('data', 'c', 'nested')} = {values(obj['c']['nested'])}",
+        values,
+    )
+    assert row.data == obj
+
+    values = QueryValues()
+    row = await db.fetchone(
+        f"SELECT * FROM test WHERE {values.json_path('data', 'b')} != {values(obj['b'])}",
```

Comment:
- the test passes also for the query ` "SELECT * FROM test",`
- more rows to the `test` table are required in order to make this test meaningful